src/System Application/App/AI/src/Copilot/CopilotCapEarlyPreview.Page.al

```diff
@@ -16,7 +16,7 @@ page 7770 "Copilot Cap. Early Preview"
     Editable = false;
     Extensible = false;
     SourceTable = "Copilot Settings";
-    SourceTableView = where(Availability = const("Early Preview"));
+    SourceTableView = where(Availability = const("Early Preview"), "Service Type" = const("Azure AI Service Type"::"Azure OpenAI"));
```

Comment:
Will the DI capabilities be exposed somewhere else to allow admin control over them?